.mk/k8s.mk

```diff
@@ -125,6 +125,9 @@ k8s-build: $(addsuffix -build,$(addprefix k8s-,$(DEPLOYS)))
 .PHONY: k8s-nsm-coredns-save
 k8s-nsm-coredns-save:  $(addsuffix -save,$(addprefix ${CONTAINER_BUILD_PREFIX}-,nsm-coredns))
 
+.PHONY: k8s-nsm-coredns-build
+k8s-nsm-coredns-build:  ${CONTAINER_BUILD_PREFIX}-nsm-coredns-build
```

Comment:
Should we use $(addsuffix -build,$(addprefix ${CONTAINER_BUILD_PREFIX}-,nsm-coredns-build for consistent?

Reply:
Indeed 👍